File: java/ej9/PruebaMovil.java
package programasPoo;

public class PruebaMovil {

  public static void main(String[] args) {
    Movil m1= new Movil("663654985", "rata");
    Movil m2= new Movil("659863214", "bisonte");
    
    System.out.println(m1);
    System.out.println(m2);
    
    m1.llama(m2, 158);
    m2.llama(m1, 169);
    
    System.out.println(m1);
    System.out.println(m2);
    
  }
}

File: python/ej9/PruebaMovil.py
'''
Prueba de la clase movil

@author: Rafael Infante
'''
from programasPoo.Movil import Movil

'''
instancias
'''
m1=Movil("663654985","rata") #0.06
m2=Movil("664665063","bisonte") #0.12
m3=Movil("66512972","mono") #0.30

'''print(m1)
print(m2)
print(m3)'''

m1.llama(m2, 1989);
m2.llama(m1, 2365);
m3.llama(m1, 2369)

print(m1)
print(m2)
print(m3)



